controls/settings/update.go

```diff
@@ -75,6 +75,17 @@ func (m Model) handleAdvancedUpdate(msg tea.Msg) (Model, tea.Cmd) {
 	return m, cmd
 }
 
+func (m Model) handleAlphaUpdate(msg tea.Msg) (Model, tea.Cmd) {
+	var cmd tea.Cmd
+	m.Alpha, cmd = m.Alpha.Update(msg)
+
+	if m.Alpha.ShouldUnfocus {
+		return m.handleSettingsUpdate(msg)
```

Comment:
I'm seeing a small issue where, if you toggle between Yes / No in the Alpha settings and then navigate up, the Settings menu still things the Alpha menu is active despite it losing focus. Someday I'll come up with a better way of managing these focus / active states, but for now you should be able to fix it with this: 
```suggestion
	    m.active = None
		return m.handleSettingsUpdate(msg)
```

Reply:
You are correct about the output size changing due to whitespace removal. I've put it back, so now sizes stay the same, but the output files are full of, IMO, unnecessary white space. Perhaps it's more user friendly this way, but I would maybe add a note to the release in the same area as your compression recommendation that, if you're using this for any sort of active deployment, removing the extra whitespace will save you some bytes.

The new test image is my own drawing of my jeep, so there aren't any licensing issues. I hear-by grant you full usage :) Or you can replace it if you hate it.